library/kotlin/io/envoyproxy/envoymobile/PulseClient.kt

```diff
@@ -41,4 +42,14 @@ interface PulseClient {
    *          It tracks the quantile stats of values.
    */
   fun distribution(vararg elements: Element, tags: Tags): Distribution
+
+  /**
+   * Logs with level, message and tags.
+   *
+   * @param level Level of the log
+   * @param message The log message
+   * @param tags Log tags, carries miscellaneous information about the log, such as file name, line
+   *             number etc.
+   */
+  fun log(level: LogLevel, message: String, tags: Map<String, String>?)
```

Comment:
Would it make sense to use `Tags` here as well? If not, would it make sense to call this something other than tags?

Reply:
it's a difficult question haha

conceptually they are kind of different but the same - `Tags` currently are used in stats APIs, and there are certain expectations for stats "tags", such as: tag names and values are supposed to be short(er) and are supposed to be more limited in size (number of tags per stats).

but the data structure for stats tags and log tags are exactly the same

i think just the sake of not having consumers to mix these two concepts (stats tags and log tags) together, it's better to make the log tags some different

i'll rename the parameter to `logTags` lmk

Reply:
Sounds good. Thanks for the explanation!